### PR TITLE
test github actions

### PR DESCRIPTION
**Pull request by richard-sl**

### CODE REVIEW

**Comment by richard-sl** [comment]
@wswebcreation https://saucelabs.github.io/sample-app-web/ :tada: 

If that's good with your latest changes I'll revert the commit that has this deploying from pull requests to just be master.

I would like to add one more commit to specify CNAME file in the deployed branch release so that we can file an OPS ticket to make this deployment live for www.saucedemo.com

We are currently changing domain registrars so OPS may have a preferred order of operations.

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build 
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
```
Comment (wswebcreation) [inline .github/workflows/github-pages.yml:17]: Can we change it to the latest stable, meaning 12?

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [10.x]
```
Comment (wswebcreation) [inline .github/workflows/node.js.yml:19]: Can we change it to the latest stable, meaning 12?

File: .github/workflows/github-pages.yml (hunk 1 above)
Comment (richard-sl) [reply .github/workflows/github-pages.yml:17]: I was getting build issues with anything other than 10 :(

I think we should tackle a node environment update separately from CICD effort

**Comment by richard-sl** [comment]
> Looks good, 2 small comments. Last but not least, do we also need to have or a process described in the README.md or on our internal wiki?

Good point I will include a README update before merging this PR to inform that merges to master will automatically deploy to production.

Another thought I have had that I would like to test from this PR is around caching. I will check the cache-control headers of the index.html going through this GitHub Pages server to get an idea of how long a release may be stale.

File: .github/workflows/github-pages.yml (hunk 1 above)
Comment (wswebcreation) [reply .github/workflows/github-pages.yml:17]: Sure, no problem

**Comment by wswebcreation** [comment]
@richard-sl 

I was checking the main url https://saucelabs.github.io/ and that is the open source page. If they publish a new version, will ours then be deleted?

**Comment by richard-sl** [comment]
@wswebcreation It should not. The website at the root of https://saucelabs.github.io is rendered from this repository: https://github.com/saucelabs/saucelabs.github.io. This repo gets deployed to a sub-folder https://saucelabs.github.io/sample-app-web/. I have to suspect that GitHub itself has some protections to avoid a naming conflict between published repos and folders within a [org-name].github.io site.

**Pull request merged**